File: youlexuan_parent/youlexuan_seckill_web/src/main/webapp/js/service/seckillGoodsService.js
//seckill_goods服务层
app.service('seckillGoodsService', function($http){
	// 查询实体
	this.findList=function () {
		return $http.get('/seckillGoods/findList.do');
	}
	//查询单体
	this.findOne=function(id){
		return $http.get('/seckillGoods/findOne.do?id='+id);
	}
});

File: youlexuan_parent/youlexuan_seckill_interface/src/main/java/com/offcn/seckill/service/SeckillGoodsService.java
package com.offcn.seckill.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbSeckillGoods;

import java.util.List;

/**
 * seckill_goods服务层接口
 * @author senqi
 *
 */
public interface SeckillGoodsService {
    List<TbSeckillGoods> findList();

    TbSeckillGoods findOne(Long id);
}
